src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -107,23 +107,46 @@ method java.lang.Object equals java.lang.Object
 method java.lang.Object getClass
 method java.lang.Object hashCode
 method java.lang.Object toString
+new java.lang.String
+staticField java.lang.String CASE_INSENSITIVE_ORDER
+new java.lang.String byte[]
+new java.lang.String byte[] int int
+new java.lang.String byte[] int int java.lang.String
+new java.lang.String byte[] int int java.nio.charset.Charset
+new java.lang.String byte[] java.lang.String
+new java.lang.String byte[] java.nio.charset.Charset
+new java.lang.String char[]
+new java.lang.String char[] int int
+method java.lang.String codePointAt int
+method java.lang.String codePointBefore int
+method java.lang.String concat java.lang.String
 method java.lang.String contains java.lang.CharSequence
+method java.lang.String contentEquals java.lang.CharSequence
+method java.lang.String contentEquals java.lang.StringBuffer
```

Comment:
Similarly, `StringBuffer` doesn't seem to be allowed? Why add this then?

Reply:
I just did not notice, but I can add methods related to it (same for `StringBuilder`). In general, leaving out methods where operations related to the argument types are not yet whitelisted gets confusing, since we have to remember to go back and add those methods whenever we add whitelist entries for new types, so I just try to add all safe methods defined on a receiver at the same time, even if we haven't whitelisted methods related to the argument types.

Reply:
I added methods defined in `AbstractStringBuilder` and constructors for `StringBuilder` and `StringBuffer` in https://github.com/jenkinsci/script-security-plugin/pull/310/commits/abd7cd362ba160851d0d33b9328a02caa5ee7b37. There is no reason to ever use `StringBuffer` instead of `StringBuilder` in the sandbox, but since it is safe, I think it makes sense to include it here anyway in case someone happens to use it.